src/test/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpointIT.java

```diff
@@ -153,4 +154,70 @@ private void createIACStub() throws IOException {
                     .withHeader("Content-Type", "application/json")
                     .withBody("[\"grtt7x2nhygg\"]")));
   }
+
+  /**
+   * Sends a sample unit in a message so that casesvc creates a case, then waits for a message on
+   * the case lifecycle queue which confirms case creation
+   *
+   * @return a new CaseNotification
+   */
+  private CaseNotification sendCaseWaitForNotification(
+      String sampleUnitRef, String sampleUnitType, UUID sampleUnitId) throws Exception {
+    createIACStub();
+
+    SimpleMessageSender sender = getMessageSender();
+
+    SampleUnitParent sampleUnit = new SampleUnitParent();
+    sampleUnit.setCollectionExerciseId(UUID.randomUUID().toString());
+    sampleUnit.setId(sampleUnitId.toString());
+    sampleUnit.setActionPlanId(UUID.randomUUID().toString());
+    sampleUnit.setSampleUnitRef(sampleUnitRef);
+    sampleUnit.setCollectionInstrumentId(UUID.randomUUID().toString());
+    sampleUnit.setSampleUnitType(sampleUnitType);
+
+    JAXBContext jaxbContext = JAXBContext.newInstance(SampleUnitParent.class);
+    String xml =
+        convertObjectToXml(
+            jaxbContext, sampleUnit, "casesvc/xsd/inbound/SampleUnitNotification.xsd");
+
+    sender.sendMessage("collection-inbound-exchange", "Case.CaseDelivery.binding", xml);
+
+    SimpleMessageListener listener = getMessageListener();
+    BlockingQueue<String> queue =
+        listener.listen(
+            SimpleMessageBase.ExchangeType.Direct,
+            "case-outbound-exchange",
+            "Case.LifecycleEvents.binding");
+
+    String message = queue.take();
+    log.info("message = " + message);
+    assertNotNull("Timeout waiting for message to arrive in Case.LifecycleEvents", message);
+    jaxbContext = JAXBContext.newInstance(CaseNotification.class);
+    return (CaseNotification)
+        jaxbContext.createUnmarshaller().unmarshal(new ByteArrayInputStream(message.getBytes()));
+  }
+
+  private static void setUnirestMapper() {
```

Comment:
Use this instead: https://github.com/ONSdigital/rm-common-test-framework/commit/5f67008b5758013f89fccaebc8958853b328cce0